src/settings/Models/FinePolicy/FinePolicy.js

```diff
@@ -33,7 +33,8 @@ export default class FinePolicy {
 
   hasValue(pathToValue) {
     const value = get(this, pathToValue);
-    return value > 0;
+
+    return !!parseFloat(value, 10);
```

Comment:
`parseValue("0.00")` will return false. Given it used to reject 0 or any negative value this may not be wrong necessarily, but it means the title remains misleading, i.e. previously it should have been `hasPositiveValue` and now it should be `hasNonZeroValue`. 

This change seems disconnected from the description in the PR. What is the goal with this change?